notebooks/demo/nx_cugraph_demo.ipynb

```diff
@@ -18,8 +18,8 @@
     "\n",
     "## System Requirements\n",
     "Using `nx-cugraph` with this notebook requires the following: \n",
-    "- NVIDIA GPU, Pascal architecture or later\n",
-    "- CUDA 12.0 - 12.9\n",
+    "- NVIDIA GPU, Volta architecture or later\n",
+    "- CUDA 12.2 - 12.9\n",
```

Comment:
In [the RAPIDS docs]( https://docs.rapids.ai/install/#system-req ), we state Pascal was dropped in RAPIDS 24.02 and that Volta is currently supported. So updated that here at the same time